### PR TITLE
Fix SSR exception issue

### PR DESCRIPTION
**Pull request by Umerbhat**
Fixed window exception issue on doing SSR

### CODE REVIEW

File: src/js/paackSvgIconSprite.js [hunk 1]
```diff
@@ -11,5 +11,7 @@ class PaackSvgIconSprite extends HTMLElement {
     this.innerHTML = svg;
   }
 }
+if(typeof window === 'undefined'){
```
Comment (PedroHLC) [inline src/js/paackSvgIconSprite.js:14]: are you sure that's what you want? `typeof window` always returns `"object"` for me...

File: src/js/paackSvgIconSprite.js [hunk 2]
```diff
@@ -11,5 +11,7 @@ class PaackSvgIconSprite extends HTMLElement {
...
+  window.customElements.define('paack-svg-icon-sprite', PaackSvgIconSprite);
+}
```
Comment (3imed-jaberi) [inline src/js/paackSvgIconSprite.js:16]: What do you think to export function to define this Web Component like this; 
```js
const definePaackSvgIconSprit = () => window.customElements.define('paack-svg-icon-sprite', PaackSvgIconSprite);
```
I think it gives us more flexibility to handle/support SSR, ISR, SSG ... outside of this module [ELM/React].

Also, we should update the condition of the current if statement;

```suggestion
if(typeof window !== 'undefined') window.customElements.define('paack-svg-icon-sprite', PaackSvgIconSprite);
```

**Review by 3imed-jaberi — CHANGES_REQUESTED** [review]

File: src/js/paackSvgIconSprite.js (hunk 2 above)
Comment (PedroHLC) [reply src/js/paackSvgIconSprite.js:16]: for me, that's fine.

Comment (mtzfactory) [reply src/js/paackSvgIconSprite.js:16]: Sounds good for me...

Comment (Umerbhat) [reply src/js/paackSvgIconSprite.js]: @3imed-jaberi I have updated missing not(!) 😓

**Review by mtzfactory — APPROVED** [review]

**Review by 3imed-jaberi — COMMENTED** [review]
@mtzfactory @PedroHLC @Umerbhat;
Let's make a release here based on this patch update.
One day when we like to change the behavior release a major version by exporting a function to define the element like this; 

```js
const definePaackSvgIconSprit = () => window.customElements.define('paack-svg-icon-sprite', PaackSvgIconSprite);
```

or export only the Web Component class and handle the definition process outside this module which gives us more flexibility to handle the side behaviors (this option is better for me).

**Review by 3imed-jaberi — APPROVED** [review]

**Pull request merged**